app/service/sorting.go

```diff
@@ -129,7 +129,7 @@ func applyOrder(query *database.DB, usedFields []string, acceptedFields map[stri
 // parsePagingParameters returns a slice of values provided for paging in a request URL (none or all should be present)
 // The values are in the order of the 'usedFields' and converted according to 'FieldType' of 'acceptedFields'
 func parsePagingParameters(r *http.Request, usedFields []string,
-	acceptedFields map[string]*FieldSortingParams) ([]interface{}, error) {
+	acceptedFields map[string]*FieldSortingParams, fieldsDirections map[string]int) ([]interface{}, error) {
```

Comment:
Probably this remark was more for the previous PR but I would not encode directions to 1/-1 explicitly but to a real enum with "asc", "desc" and maybe "unknown", and so the auth values would be one of these... (you would even be able to create a method on this type to return you the SQL direction as a string 😎).

Reply:
done

Reply:
👍🙂